### ISSUE TITLE
Advanced Visualization Projects

### ISSUE THREAD

**Issue by denibulkashvili** [chingu-voyages/v11-bears-team-16#4]
For example, an app that calculates and animates how path-finding algorithms (e.g.  A* or Dijkstra's) find a path between two points. Alternatively, visualization of sorting algorithms. So basically the user can play around with arrays, chose an algorithm and see it being sorted in real time. 
5

**Comment by willjw3** [comment]
I'm assuming you mean something a little more involved than this. Am I right?
https://www.freecodecamp.org/forum/t/bubble-sort-visualizer/311738

**Comment by denibulkashvili** [comment]
Yes, exactly. What I had in mind is more interactive where the user can change the array itself, and can choose different algorithms
